packages/next-preact/package.json

```diff
@@ -9,7 +9,8 @@
     "preact-compat": "^3.17.0"
   },
   "dependencies": {
-    "module-alias": "2.0.6"
+    "module-alias": "2.0.6",
+    "preact-context": "^1.1.3"
```

Comment:
Can we please make this a peer dependency?

Reply:
OK, I added it to peer dependency.